e2e/alerting_test.go

```diff
@@ -159,15 +172,23 @@ func TestCoolDown(t *testing.T) {
 		return height != nil && height.Uint64() > receipt.BlockNumber.Uint64(), nil
 	}))
 
+	time.Sleep(1 * time.Second)
```

Comment:
Why are we sleeping here? The requirement loop should ensure that pessimism is synced with the latest block height

Reply:
Yeah was testing something and forgot to delete this

Reply:
[74f2414](https://github.com/base-org/pessimism/pull/200/commits/74f241413764e3b9caf17a604ffe27f2bc28480c)